frontend/src/citizen-frontend/calendar/CalendarPage.tsx

```diff
@@ -181,28 +186,61 @@ const CalendarPage = React.memo(function CalendarPage() {
                 </ContentArea>
               </RenderOnlyOn>
               <RenderOnlyOn desktop>
-                <CalendarGridView
-                  childData={response.children}
-                  calendarDays={response.days}
-                  onCreateReservationClicked={
-                    openReservationModalWithoutInitialRange
-                  }
-                  onCreateAbsencesClicked={(date) =>
-                    openAbsenceModal(date, false)
-                  }
-                  onOpenDiscussionReservationsClicked={
-                    openDiscussionSurveyModal
-                  }
-                  onReportHolidaysClicked={openHolidayModal}
-                  selectedDate={
-                    modalState?.type === 'day' ? modalState.date : undefined
-                  }
-                  selectDate={openDayModal}
-                  includeWeekends={true}
-                  dayIsReservable={dayIsReservable}
-                  events={events}
-                  isDiscussionActionVisible={showDiscussions}
-                />
+                {featureFlags.calendarMonthView && (
+                  <CalendarToggle
+                    value={overrideToUseCalendarList}
+                    onClick={() =>
+                      setOverrideToUseCalendarList(!overrideToUseCalendarList)
+                    }
+                  />
+                )}
+                {featureFlags.calendarMonthView && overrideToUseCalendarList ? (
```

Comment:
Pitäisi kai olla vakiona tämä kuukausiversio käytössä, ellei ole manuaalisesti vaihdettu listanäkymää päälle
```suggestion
                {featureFlags.calendarMonthView && !overrideToUseCalendarList ? (
```